Task 9

misostc opened the pull request.

misostc committed:
Task 9

vandus reviewed (changes requested):
Takisto odoberanie filmov  nefunguje.
Ked film pridam, button zmizne a uz ho nemozem odobrat.
KEd idem do saved filmov, buttonm je tam znovu a ked na neho kliknem , asi to chce znovu pridat ten film miesto odobrania
 E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #2
                                                                                                    Process: cz.muni.fi.pv256.movio2.uco_410434.alternative.debug, PID: 8011
                                                                                                    java.lang.RuntimeException: An error occurred while executing doInBackground()
                                                                                                        at android.os.AsyncTask$3.done(AsyncTask.java:353)
                                                                                                        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
                                                                                                        at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
                                                                                                        at java.util.concurrent.FutureTask.run(FutureTask.java:271)
                                                                                                        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)
                                                                                                        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
                                                                                                        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
                                                                                                        at java.lang.Thread.run(Thread.java:764)
                                                                                                     Caused by: android.database.SQLException: Failed to insert row into content://cz.muni.fi.pv256.movio2.uco_410434/film
                                                                                                        at cz.muni.fi.pv256.movio2.uco_410434.data.FilmProvider.insert(FilmProvider.java:105)
                                                                                                        at android.content.ContentProvider$Transport.insert(ContentProvider.java:267)
                                                                                                        at android.content.ContentResolver.insert(ContentResolver.java:1539)
                                                                                                        at cz.muni.fi.pv256.movio2.uco_410434.data.PersistedFilmManager.createFilm(PersistedFilmManager.java:38)

misostc commented:
Ahoj, zradilo ma, že som si to neotestoval s rebuildom 😢 , fixnem aj task8 aj tak9 do jedného commitu

misostc committed:
Task 8 & Task 9 fixes

- fixed image too big problems (downloading smaller version from server)
- fixed persistence problems (forgot to store id in parcel)
- fixed tablet landscape problem

misostc committed:
Add missing translation
